packages/nextjs/test/integration/next13.appdir.config.template

```diff
@@ -5,7 +5,7 @@ const moduleExports = {
     ignoreDuringBuilds: true,
   },
   experimental: {
-    appDir: Number(process.env.NODE_MAJOR) >= 16, // experimental.appDir requires Node v16.8.0 or later.
+    appDir: true, // experimental.appDir requires Node v16.8.0 or later.
```

Comment:
l: why this change?

Reply:
Oh I missed that. Just a leftover from troubleshooting. Thanks!